pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -173,21 +170,25 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
           </div>
 
           <div className={styles.cardsContainer}>
-            <OrganizationCard
-              title={{ label: organization.name, href: `/organizations/${organization.id}` }}
-              description={organization.description}
-              website={organization.website}
-              logo={organization.logo}
-              components={{
-                owned: organization.owns?.length.toString() ?? "0",
-                supported: organization.supports?.length.toString() ?? "0",
-                used: organization.uses?.length.toString() ?? "0",
-              }}
-              gitHub={organization.github}
-              gitLab={organization.gitlab}
-              type={organization.type}
-              layoutClassName={styles.organizationCardContainer}
-            />
+            {organizationData ? (
```

Comment:
To get rid of this ternary expression we should use 
`{_getComponent?.data?.embedded?.url?.embedded?.organisation && (...)}` if it exists and 
`{!_getComponent?.data?.embedded?.url?.embedded?.organisation && (...)}` if it does not exist. 

We should do this for better readability and code consistency